src/riak_dt_map.erl

```diff
@@ -299,6 +303,9 @@ value(_, Map) ->
 %% Atomic, all of `Ops' are performed successfully, or none are.
 -spec update(map_op(), riak_dt:actor() | riak_dt:dot(), map()) ->
                     {ok, map()} | precondition_error().
+update(Op, ActorOrDot, {_C, V, D}=Map) when is_list(V);
```

Comment:
It's inconsisent with `value/1` that we check `is_list/1` on both values and deferred ops here. It's also unnecessary since that is checked in `to_v2/1`.